docs/MagicStringDetector.md

```diff
@@ -0,0 +1,122 @@
+# Magic String Detector
+
+This command helps to find Magic Strings in your code or in specific php language constructs (like just checking switch cases etc.). It can also ignore empty strings (if you want to). 
```

Comment:
```suggestion
This command helps to find Magic Strings in your code or in specific php language constructs (like just checking switch cases etc.). It can also ignore empty strings (if you want it to). 
```